backends/p4tools/BUILD.bazel

```diff
@@ -0,0 +1,138 @@
+# The BUILD file for p4testgen, inspired by the tool's cmake build rules.
+
+exports_files(["p4tools.def"])
+
+package(
+    #default_applicable_licenses = ["//:license"],
+    licenses = ["notice"],
+)
+
+# Extend the list with the name of the desired targets (in "modules/testgen/targets/).
+TESTGEN_TARGETS = ["bmv2"]
```

Comment:
Nit: since we expect this to be extended, add a trailing comma to the list and make it multi-line?
```
TESTGEN_TARGETS = [
  "bmv2",
]
```